src/services/config.js

```diff
@@ -8,7 +8,13 @@ config.parseServerConfig()
 config.parseDatabaseConfig()
 config.parseKeyConfig()
 
-if (process.env.NODE_ENV === 'unit') {
+function isRunningUnitTests () {
```

Comment:
This is a minor point, but I would argue that the tests are not unit tests, since they test the http endpoints, rather than individual methods. With this in mind, I think this function could be renamed to `isRunningTests`. Aside from that, LGTM.